Content.Server/GameObjects/Components/Power/PowerDevice.cs

```diff
@@ -193,7 +207,7 @@ private void UpdateLoad(float value)
             if (Connected == DrawTypes.Node)
             {
                 var node = Owner.GetComponent<PowerNodeComponent>();
-                node.Parent.UpdateDevice(this, oldLoad);
+                node.Parent?.UpdateDevice(this, oldLoad);
```

Comment:
This is a pretty bad fix. Node parent shouldn't be null if the draw type is node.

I made a proper fix to your branch: https://github.com/DamianX/space-station-14/pull/1